frontend/src/screens/wallet/send/Onchain.tsx

```diff
@@ -0,0 +1,234 @@
+        );
+      }
+      setLoading(true);
+      if (isSwap) {
+        const swapOutResponse = await request<SwapResponse>("/api/swaps/out", {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({
+            swapAmount: +amount,
+            destination: address,
+          }),
+        });
+        if (!swapOutResponse) {
+          throw new Error("Error swapping out");
+        }
+        navigate(`/wallet/swap/out/status/${swapOutResponse.swapId}`);
+        toast({ title: "Initiated swap" });
+      } else {
+        const response = await request<RedeemOnchainFundsResponse>(
+          "/api/wallet/redeem-onchain-funds",
+          {
+            method: "POST",
+            headers: {
+              "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+              toAddress: address,
+              amount: +amount,
+              feeRate: +feeRate,
+            }),
+          }
+        );
+        if (!response?.txId) {
+          throw new Error("No address in response");
+        }
+        navigate(`/wallet/send/onchain-success`, {
+          state: {
+            amount: +amount,
+            txId: response.txId,
+          },
+        });
+        toast({
+          title: "Successfully broadcasted transaction",
+        });
+      }
+    } catch (e) {
+      toast({
+        variant: "destructive",
+        title: "Failed to send payment",
+        description: "" + e,
+      });
+      console.error(e);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  React.useEffect(() => {
+    if (!address) {
+      navigate("/wallet/send");
+    }
+  }, [navigate, address]);
+
+  if (!balances || !address || (!recommendedFees && !mempoolError)) {
+    return <Loading />;
+  }
+
+  return (
+    <form onSubmit={onSubmit} className="grid gap-6 md:max-w-lg">
+      <div className="grid gap-2">
+        <Label>Recipient</Label>
+        <div className="flex flex-wrap gap-2 items-center">
+          {address.match(/.{1,4}/g)?.map((word, index) => {
+            if (index % 2 === 0) {
+              return (
+                <span key={index} className="text-foreground">
+                  {word}
+                </span>
+              );
+            } else {
+              return (
+                <span key={index} className="text-muted-foreground">
+                  {word}
+                </span>
+              );
+            }
+          })}
+        </div>
+      </div>
+      <div className="grid gap-2">
+        <Label htmlFor="amount">Amount</Label>
+        <Input
+          id="amount"
+          type="number"
+          value={amount}
+          placeholder="Amount in Satoshi..."
+          onChange={(e) => {
+            setAmount(e.target.value.trim());
+          }}
+          min={isSwap ? 50000 : 1}
```

Comment:
```suggestion
          min={isSwap ? 50000 : 330}
```

Reply:
I'm using `ONCHAIN_DUST_SATS` instead of `330` but it is set to `1000`, is that okay?